### PR TITLE
Refined import db process

### PR DESCRIPTION
**Pull request by iulianpascalau**
Improved the import-db process:
- added a delay before the import process will stop the node in order to have the time to commit received data
- added ThresholdMinConnectedPeers as a config in order to be overridden by the import db process
- made the import-db process automatically generate a new BLS key in order to avoid accidental usage of a validator BLS key that migh trigger the node shuffle-out

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

File: cmd/node/main.go [hunk 1]
```diff
@@ -1491,17 +1494,17 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
...
 }
 
-func applyCompatibleConfigs(log logger.Logger, config *config.Config, ctx *cli.Context) {
-	importDbDirectoryValue := ctx.GlobalString(importDbDirectory.Name)
-	if len(importDbDirectoryValue) > 0 {
+func applyCompatibleConfigs(isInImportMode bool, log logger.Logger, config *config.Config, p2pConfig *config.P2PConfig) {
+	if isInImportMode {
 		importCheckpointRoundsModulus := uint(config.EpochStartConfig.RoundsPerEpoch)
-		log.Info("import DB directory is set, altering config values!",
+		log.Warn("the node is import mode! Will auto-set some config values",
```
Comment (SebastianMarian) [inline cmd/node/main.go:1500]: is in

File: factory/cryptoSigningParams.go [hunk 2]
```diff
@@ -78,6 +91,21 @@ func (cspf *cryptoSigningParamsLoader) Get() (*CryptoParams, error) {
 	return cryptoParams, nil
 }
 
+func (cspf *cryptoSigningParamsLoader) generateCryptoParams(cryptoParams *CryptoParams) (*CryptoParams, error) {
+	log.Warn("the node is import mode! Will generate a fresh new BLS key")
```
Comment (SebastianMarian) [inline factory/cryptoSigningParams.go:95]: is in

**Review by SebastianMarian — DISMISSED** [review]

File: cmd/node/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/main.go:1500]: done

File: factory/cryptoSigningParams.go (hunk 2 above)
Comment (iulianpascalau) [reply factory/cryptoSigningParams.go]: done

**Review by SebastianMarian — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed.

**Pull request merged**